docs/README.md

```diff
@@ -1,18 +1,19 @@
+
 # Xen Orchestra
 
 ## Introduction
 
-Welcome on the official Xen Orchestra (XO) documentation.
+Welcome to the official Xen Orchestra (XO) documentation.
 
-XO is a web interface to visualize and administrate your XenServer (or XAPI enabled) hosts. **No agent** is requiered to make it work.
+XO is a web interface to visualize and administer your XenServer (or XAPI enabled) hosts. **No agent** is required for it to work.
```

Comment:
http://site.uit.no/english/words/administrate/

Did not know that, thanks :slightly_smiling_face: